### DISCUSSION TITLE
Constrain Crop

### DISCUSSION THREAD

**Discussion by cjbaur23** [rsjaffe/MIDI2LR#568] [Ideas]
Is there a possibility to use the "Constrain Crop" feature from the Transform-panel in the Develop module? I haven't found a shortcut for this feature either.

**Comment by rsjaffe** [comment]
Currently, it's misplaced in Lens corrections group -> Constrain to warp. Assign this to a key and it will work. This toggles constrain to picture in both the crop panel and the transform panel. I'll move the command to the crop group next release (Lightroom moved things around in previous releases and I failed to move this particular command at the same time).

**Reply by rsjaffe** [reply]
Actually, will move to transform group.

**Reply by cjbaur23** [reply]
Thanks a lot - you made my day!
